### PR TITLE
wrap post response correctly

### PR DESCRIPTION
**Pull request by NathanielBennett**
Fixes the json response to updating saved articles so that it matches that currently provided by id.guardianapis.com, We post to /syncedPrefs/me/savedArtices. we GET from /syncedPrefs/me and the generated json needs to reflect this, MY BAD

Also changes the response headers so that they match those currentty live.

### CODE REVIEW

**Comment by NathanielBennett** [comment]
@alexduf - Yes I checked.

File: src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala [hunk 1]
```diff
@@ -17,7 +17,7 @@ class FetchArticlesController(fetchSavedArticles: FetchSavedArticles)(implicit e
          syncedPrefs.savedArticles.map( sp =>
```
Comment (markrichardsg) [inline src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala]: I find foreach assists reading as it's obvious that the return isn't intended to be used

File: src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala [hunk 2]
```diff
@@ -17,7 +17,7 @@ class FetchArticlesController(fetchSavedArticles: FetchSavedArticles)(implicit e
...
             logger.debug(s"Returning found: ${sp.articles.size} articles")
          )
-         Future { okSavedArticlesResponse(syncedPrefs) }
+         Future { okSycedPrefsResponse(syncedPrefs) }
```
Comment (markrichardsg) [inline src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala]: This can be a Future.successful

Comment (markrichardsg) [inline src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala]: Given the change in response, should the controller be renamed to match (FetchArticlesController -> FetchSyncedPrefsController)

File: src/main/scala/com/gu/sfl/controller/SaveForLaterController.scala [hunk 3]
```diff
@@ -13,5 +13,6 @@ trait SaveForLaterController {
   val serverErrorResponse = LambdaResponse(StatusCodes.internalServerError, Some("Server error."))
   val accessDenied = LambdaResponse(StatusCodes.forbidden, Some("Access denied"))
   val emptyArticlesResponse = LambdaResponse(StatusCodes.ok, Some(mapper.writeValueAsString(SavedArticles(List.empty))))
-  def okSavedArticlesResponse(syncedPrefs: SyncedPrefs): LambdaResponse = LambdaResponse(StatusCodes.ok, Some(mapper.writeValueAsString(SyncedPrefsResponse("ok", syncedPrefs))))
+  def okSycedPrefsResponse(syncedPrefs: SyncedPrefs): LambdaResponse = LambdaResponse(StatusCodes.ok, Some(mapper.writeValueAsString(SyncedPrefsResponse("ok", syncedPrefs))))
```
Comment (markrichardsg) [inline src/main/scala/com/gu/sfl/controller/SaveForLaterController.scala]: typo okSyc... -> okSync...

Comment (NathanielBennett) [reply src/main/scala/com/gu/sfl/controller/SaveForLaterController.scala]: aaargh and I f*cked it up anwayay

File: src/main/scala/com/gu/sfl/lambda/LambdaApiGateway.scala [hunk 4]
```diff
@@ -60,7 +60,8 @@ object LambdaResponse extends Base64Utils {
 case class LambdaResponse(
   statusCode: Int,
   maybeBody: Option[String],
-  headers: Map[String, String] = Map("Content-Type" -> "application/json"))
+  headers: Map[String, String] = Map("Content-Type" -> "application/json; charset=UTF-8", "cache-control" -> "max-age=0")
```
Comment (markrichardsg) [inline src/main/scala/com/gu/sfl/lambda/LambdaApiGateway.scala]: HTTP headers are supposed to be case insensitive, so it should not matter.
But, sometimes clients/servers get it wrong: is cache-control all lower case okay on both IOS and Android?

Comment (NathanielBennett) [reply src/main/scala/com/gu/sfl/lambda/LambdaApiGateway.scala]: The current service uses this form, so I think it's safe to assume this is fine for all clients

File: cfn.yaml [hunk 5]
```diff
@@ -37,7 +37,7 @@ Mappings:
       TableWriteCapacity: 25
     PROD:
       TableReadCapacity: 75
-      TableWriteCapacity: 75
+      TableWriteCapacity: 4500
```
Comment (alexduf) [inline cfn.yaml]: That shouldn't be in the cloudformation :)

**Pull request merged**